package.json

```diff
@@ -18,5 +18,9 @@
   },
   "devDependencies": {
     "webpack-dev-server": "^2.11.1"
+  },
+  "volta": {
```

Comment:
Out of curiosity, what is this `volta` section about?

Reply:
it's js tooling, volta.sh